backend/tessend/tessend.go

```diff
@@ -42,6 +42,32 @@ const (
 	// ringBackendKeepalive is the length of time, in seconds, that the
 	// ring considers an entry alive.
 	ringBackendKeepalive = 900
+
+	// perResourceDuration is the duration of time, in seconds, that TessenD will
+	// wait in between resources when collecting its respective count.
+	perResourceDuration = 5 * time.Second
+)
+
+var (
+	// resourceMetrics maps the metric name to the etcd function
+	// responsible for retrieving the resources store path.
+	resourceMetrics = map[string]func(context.Context, string) string{
```

Comment:
A bit off topic but reading through this list just made me realize it would be cool to have some metrics about enterprise resources. Do you know if that's something planned?

Reply:
That's a really good question. I don't think we explicitly talked about this. @annaplotkin @portertech thoughts?

I suppose if we did want enterprise resource metrics, we would have to export this field and append any enterprise store paths of interest.

Reply:
@nikkiki We haven't discussed the Enterprise portion specifically but since folks with licenses are required to keep Tessen on, it would make sense 👍 Would love to hear @portertech thoughts as well.

Reply:
https://github.com/sensu/sensu-go/issues/2905#issuecomment-491410646